tests/MongoDB.Driver.Examples/DocumentationExamples.cs

```diff
@@ -1568,6 +1568,46 @@ public void Index_Example_2()
             result.Should().Be("cuisine_1_name_1");
         }
 
+        [Fact]
+        public void Causal_Consistency_Example_2()
+        {
+            RequireServer.Check().SupportsCausalConsistency();
+
+            this.client.GetDatabase("myDatabase").DropCollection("myCollection");
+
+            // Start Tunable consistency controls example
+            var connectionString = "mongodb://localhost/?readPreference=secondaryPreferred";
```

Comment:
Can't remember if we run these tests on Evergreen. This connection string probably won't work on Evergreen.

Reply:
I think it can be checked by using evergreen command line? I will try to run this by evergreen